cirq/ops/pauli_string.py

```diff
@@ -179,6 +179,12 @@ def __str__(self):
 
         return prefix + '*'.join(factors)
 
+    def _unitary_(self) -> np.ndarray:
+        u = np.array([self.coefficient], dtype=np.complex128)
```

Comment:
Need to return `None` when the coefficient is not on the complex unit circle. Add a test checking this.